Adding more info to Fusion upgrade (#9762)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/upgrade-dbt-platform-version.md

#### Prerequisites

To take advantage of the upgrade assistant and other upgrade tools, you'll need to meet the following prerequisites:
- Your dbt project must be updated to use the **Latest** release track.
- You must have a `developer` license.

> - You must have the proper [permissions set](/docs/platform/manage-access/enterprise-permissions) to access all of the upgrade features:

Reviewer: I only see two roles that can access all of these. Should we state that each upgrade task can be performed by different people? Or one person with all of these permission sets?

> - You must have the proper [permissions set](/docs/platform/manage-access/enterprise-permissions) to access all of the upgrade features:

Author: ```suggestion
- You must have the proper [permissions set](/docs/platform/manage-access/enterprise-permissions) to execute individual upgrade tasks. Migrating to Fusion is a multi-step process and some of these steps may be repeated across projects by different users:
```

> | Feature | Required permission(s) | Permission sets with access |

Reviewer: For the "Permission sets with access" column, does it mean that
only Admins and Account Amins can perform upgrade? Or do different roles perform upgrades for  different "features?"

> | Feature | Required permission(s) | Permission sets with access |

Reviewer: Do these upgrade features all need to occur in order to be fully upgraded? Maybe this can be specifically upgrade feature or upgrade task?
```suggestion
| Upgrade task | Required permission(s) | Permission sets with access |
```

Author: Two of these - Run once on fusion and View Fusion readiness.. - are both optional steps so they don't have to go through them, but changing the column name to "upgrade task" makes sense to me. I will reorder since "Enable Fusion access" has to be done to even see Fusion updates in the account

> | Upgrade task | Required permission(s) | Permission sets with access |

Author: ```suggestion
| Upgrade task | Required permission(s) | Supported permission sets |
```

|---------|------------------------|-----------------------------|
| Enable Fusion access (triggers migration flows) | Fusion readiness: `write` <br></br> Projects: `write` | Admin, Account Admin and anyone assigned the Fusion admin set, provided their base role also has `write` access to projects |
| View Fusion readiness and job eligibility | Fusion readiness: `read` | Developer, Admin, Member, Account Admin, Fusion admin |

> | "Run once on Fusion" | Runs: `write` | Job Admin, Job Runner, Admin, Member, Project Creator, Account Admin |

Reviewer: Why is this upgrade task quoted?

Author: It was for the button action, fixed it to match style guide

> | "Run once on Fusion" | Runs: `write` | Job Admin, Job Runner, Admin, Member, Project Creator, Account Admin |

Author: ```suggestion
| **Run once on Fusion** job action | Runs: `write` | Job Admin, Job Runner, Admin, Member, Project Creator, Account Admin |
```
